CMakeLists.txt

```diff
@@ -171,7 +173,6 @@ set(LUA_FOUND true)
 message("CMake build ${CMAKE_BUILD_TYPE}, CFLAGS=${CMAKE_C_FLAGS}, CXXFLAGS=${CMAKE_CXX_FLAGS}")
 
 add_definitions(-D_MT -D_CPPUNWIND -DPURE_DYNAMIC_CAST -DDECLARE_SPECIALIZATION -DM_NOSTDCONTAINERS_EXT)
-add_definitions(-DLUABIND_DYNAMIC_LINK) #self-explanatory
```

Comment:
Luabind can be included from any engine module, so this has to be global.

Reply:
This define is present in `Externals/luabind/CMakeLists.txt:23`

Reply:
Oh, ok. And it is defined for any module too?

Reply:
Like, when you build xrEngine, -DLUABIND_DYNAMIC_LINK should be defined in xrEngine too.

Reply:
@Masterkatze

Reply:
Hmm, I think it's better to put it back for now.